projects/core/components/dialog/test/dialog.factory.spec.ts

```diff
@@ -0,0 +1,155 @@
+import * as fs from 'node:fs';
```

Comment:
@MillerSvt Maybe we can use something 

https://github.com/tsdjs/tsd or https://github.com/garronej/tsafe or `// @ts-expect-error`

what do you think?

Reply:
Hmm, I forgot about `@ts-expect-error`. I'll check it out next week.

Reply:
See what happened...